src/jarabe/journal/listview.py

```diff
@@ -747,7 +747,11 @@ def __icon_clicked_cb(self, cell, path):
 
     def __cell_title_edited_cb(self, cell, path, new_text):
         iterator = self._model.get_iter(path)
-        self._model[iterator][ListModel.COLUMN_TITLE] = new_text
+        if new_text.isspace() or new_text == '':
+            return
+        else:
+            self._model[iterator][ListModel.COLUMN_TITLE] = new_text
+
         self.emit('title-edit-finished')
```

Comment:
Oh yes, I just saw the use of the signal 'title-edit-finished' in the journalactivity.py .
I will make the function emit the required signal by adding the line self.emit(..) in if block.
Hope that would be fine.